decidim-core/app/models/decidim/participatory_process_step.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+module Decidim
+  # A ParticipatoryProcess is composed of many steps that hold different
+  # features that will show up in the depending on what step is currently
+  # active.
+  class ParticipatoryProcessStep < ApplicationRecord
+    belongs_to :participatory_process, foreign_key: "decidim_participatory_process_id", class_name: Decidim::ParticipatoryProcess
```

Comment:
Yeah I guess dates (although we'll be duplicating them at the form, and also presence of participatory_process.

Oh, and there should be a unique active step per process (maybe we need to also add a unique index or something to the migration)